docs/developer/developer-intellij-setup.mdx

```diff
@@ -69,6 +69,22 @@ While debugging, you can run code step-by-step by using the debugger tools.
 
 By clicking the play button, `lfc` will be invoked, and if compilation is successful, its output can be found, relative to package root of the file under compilation, in `bin` if the target is a compiled language (e.g., C) or in `src-gen` if the target is an interpreted language (e.g., TypeScript).  For the `HelloWorld.lf` example above, the binary can be found in `test/Cpp/bin/HelloWorld` and can be executed in the terminal.
 
+## Starting and Debugging the Language Server
+
+There should a predefined run configuration called <kbd>LanguageDiagramServer</kbd> available, which executes the main method the `LanguageDiagramServer` class, which, as a result, starts the language server.
```

Comment:
Is this actually predefined? Or does the user have to define it?

Reply:
It is predefined once [this PR](https://github.com/lf-lang/lf-lang.github.io/pull/253) gets merged. I cloned the reository once more and checked out the sdo/debug-ls branch and the run configuration does just appear as the currently selected run configuration.

Reply:
Is this ready to get merged? Note that https://github.com/lf-lang/lf-lang.github.io/pull/253 has been merged.